src/pp-annotator/viewer/PrzypisViewer.tsx

```diff
@@ -262,19 +289,22 @@ export default class PrzypisViewer extends Widget {
     //
     // Returns nothing.
     _clearHideTimer = () => {
+        if (!this.hideTimer || !this.hideTimerDfd || this.hideTimerActivity) {
+            throw new Error('Expected timer to be initialized!');
+        }
         clearTimeout(this.hideTimer);
         this.hideTimer = null;
         this.hideTimerDfd.reject();
-        this.hideTimerActivity = null;
+        this.hideTimerActivity = false;
     }
 }
 
 
 
 // Classes for toggling annotator state.
-PrzypisViewer.classes = {
+Object.assign(PrzypisViewer.classes, {
```

Comment:
static property?

Reply:
ogólnie kod który przyjmuje opcje klasy jest trochę hacky, polega na tym że konstruktor bazowy przypisze opcje do klasy a potem te opcje są rozszerzane o kolejne. Powiedziałbym że ogarnięcie tego to już osobny task.